analytics/mapreduce/src/main/java/mil/nga/giat/geowave/analytic/mapreduce/operations/KdeCommand.java

```diff
@@ -62,6 +64,29 @@ public KDEJobRunner createRunner(
 		// Config file
 		File configFile = (File) params.getContext().get(
 				ConfigOptions.PROPERTIES_FILE_CONTEXT);
+		
+		if (configFile == null && VersionUtils.getVersion() == null) {
+			final File defaultPath = ConfigOptions.getDefaultPropertyPath();
+			final String[] configFiles = defaultPath.list(
+					new FilenameFilter() {
+
+						@Override
+						public boolean accept(
+								File dir,
+								String name ) {
+							return name.endsWith("-config.properties");
+						}
+					});
+			if (configFiles != null && configFiles.length > 0) {
+				final String firstFile = String.format(
+						"%s%s%s",
+						defaultPath.getAbsolutePath(),
+						File.separator,
+						configFiles[0]);
+				configFile = new File(firstFile);
+			}
+			
+		}
```

Comment:
Just curious why this code is here ... but the code in ConfigOptions looks fairly similar?  I haven't dug deep into the code so they may be doing different things, but I would hope that we would just pass the better config file as ConfigOptions.PROPERTIES_FILE_CONTEXT instead of putting this snippet of code everywhere in the baseline?